conf/tracing.go

```diff
@@ -0,0 +1,38 @@
+package conf
```

Comment:
conf is not new. https://github.com/raintank/metrictank/tree/master/conf
before it was just used for extra config related things like storage-schemas and aggregation rules config, and now also for tracing config/setup.  I thought about putting it in a `tracing` package but it would be a tiny package. though perhaps that's the better way.